packages/react/src/components/Button/Button.Skeleton.js

```diff
@@ -23,7 +23,7 @@ const ButtonSkeleton = ({ small, href }) => {
     className: buttonClasses,
   };
 
-  const button = <button {...commonProps} type="button" />;
+  const button = <div {...commonProps} type="button" />;
```

Comment:
Would we want `type="button"` on it still as a `div`?

Reply:
It wasn't throwing any DAP errors but doesn't hurt to just use a simple `div`. Will update